specification/chaos/resource-manager/Microsoft.Chaos/preview/2023-10-27-preview/examples/PrivateAccessPrivateEndpointConnection_Approve.json

```diff
@@ -0,0 +1,55 @@
+{
+  "parameters": {
+    "subscriptionId": "6b052e15-03d3-4f17-b2e1-be7f07588291",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2023-10-27-preview",
+    "privateAccessName": "myPrivateAccess",
+    "privateEndpointConnectionName": "myPrivateEndpointConnection",
+    "privateEndpointConnection": {
+      "properties": {
+        "privateLinkServiceConnectionState": {
+          "status": "Approved",
+          "description": "Approving myPrivateEndpointConnection"
+        }
+      }
+    }
+  },
+  "responses": {
+    "201": {
+      "body": {
+        "name": "myPrivateEndpointConenction",
+        "type": "Microsoft.Chaos/privateAccesses/privateEndpointConnections",
+        "id": "/subscriptions/6b052e15-03d3-4f17-b2e1-be7f07588291/resourceGroups/myResourceGroup/providers/Microsoft.Chaos/privateAccesses/myPrivateAccess/privateEndpoinConnections/myPrivateEndpointConnectionName",
+        "properties": {
+          "provisioningState": "Updating",
+          "privateEndpoint": {
+            "id": "/subscriptions/6b052e15-03d3-4f17-b2e1-be7f07588291/resourceGroups/myResourceGroup/providers/Microsoft.Network/privateEndpoints/myPrivateEndpoint"
+          },
+          "privateLinkServiceConnectionState": {
+            "actionsRequired": "None",
+            "description": "Approving myPrivateEndpointConnection",
+            "status": "Approved"
+          }
+        }
+      }
+    },
+    "200": {
+      "body": {
+        "name": "myPrivateEndpointConnectionName",
+        "type": "Microsoft.Chaos/privateAccesses/PrivateEndpointConnections",
+        "id": "/subscriptions/6b052e15-03d3-4f17-b2e1-be7f07588291/resourceGroups/myResourceGroup/providers/Microsoft.Chaos/privateAccesses/myPrivateAccess/privateEndpoinConnections/myPrivateEndpointConnectionName",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "privateEndpoint": {
+            "id": "/subscriptions/6b052e15-03d3-4f17-b2e1-be7f07588291/resourceGroups/myResourceGroup/providers/Microsoft.Network/privateEndpoints/myPrivateEndpoint"
+          },
+          "privateLinkServiceConnectionState": {
+            "actionsRequired": "None",
+            "description": "Approving myPrivateEndpointConnection",
+            "status": "Approved"
+          }
+        }
+      }
```

Comment:
This 200 is not right - this would never be returned from this endpoint, but rather from the status endpoint returned in the header.